modules/ROOT/pages/clustering/server-syntax.adoc

```diff
@@ -0,0 +1,174 @@
+:description: This section provides a summary of server management Cypher syntax.
+[role=enterprise-edition]
+[[server-commands-reference]]
+= Server commands reference
+
+Servers can be added and managed using a set of Cypher administration commands executed against the `system` database.
+
+When connected to the DBMS over `bolt`, administration commands are automatically routed to the `system` database.
+
+
+[[server-management-syntax]]
+== Server management command syntax
+
+[NOTE]
+====
+More details about the syntax descriptions can be found xref:database-administration/syntax.adoc[here].
+====
+
+[cols="<15s,<85"]
+|===
+| Command
+m| ENABLE SERVER
+
+| Syntax
+a|
+[source, syntax, role=noheader]
+----
+ENABLE SERVER 'serverId' [OPTIONS "{" option: value[,...] "}"]
+----
+
+| Description
+a| Adds a server that has been discovered to the cluster.
+For more information see xref:clustering/servers.adoc#server-enabled-state[Enabled state].
+
+| Required privilege
+a| `GRANT SERVER MANAGEMENT`
+
+(see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-privilege-management[SERVER MANAGEMENT privileges])
+|===
+
+[cols="<15s,<85"]
+|===
+| Command
+m| ALTER SERVER
+
+| Syntax
+a|
+[source, syntax, role=noheader]
+----
+ALTER SERVER 'name' SET OPTIONS "{" option: value[,...] "}"
+----
+
+| Description
+a| Changes the constraints for a server.
+For more information see xref:clustering/servers.adoc#alter-server-options[Altering server options].
+
+| Required privilege
+a| `GRANT SERVER MANAGEMENT`
+
+(see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-privilege-management[SERVER MANAGEMENT privileges])
+|===
+
+[cols="<15s,<85"]
+|===
+| Command
+m| RENAME SERVER
+
+| Syntax
+a|
+[source, syntax, role=noheader]
+----
+RENAME SERVER 'name' TO 'newName'
+----
+
+| Description
+a| Changes the name of a server.
+For more information see xref:clustering/servers.adoc#_renaming_a_server[Renaming a server].
+
+| Required privilege
+a| `GRANT SERVER MANAGEMENT`
+
+(see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-privilege-management[SERVER MANAGEMENT privileges])
+|===
+
+[cols="<15s,<85"]
+|===
+| Command
+m| REALLOCATE DATABASES
```

Comment:
Deallocate at the moment is very much tied to a server, so that is fine here. 
But reallocate is not a server level command. It is a cluster level .. thing. I guess it needs to go somewhere. I might as well be this page, but at least could we push it to the end and separate it a bit from the rest?

Reply:
I pushed it down to the bottom and added a little note about this, but even though it isn't a server-level command, could the belief that it is cause issues? I mean, do we need to point it out?

Reply:
No "the belief" won't cause problems.